### PR TITLE
Helm v2.15 is hopelessly busted

### PR DESCRIPTION
**Pull request by sdake**
See Helm issue: https://github.com/helm/helm/issues/6708 for more
details.

Closes: https://github.com/istio/istio.io/issues/5221

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/install/helm/index.md [hunk 1]
```diff
@@ -40,7 +40,9 @@ and then further customize the configuration for your specific needs.
 
 1. Check the [Requirements for Pods and Services](/docs/setup/additional-setup/requirements/).
 
-1. [Install a Helm client](https://github.com/helm/helm#install) with a version higher than 2.10.
+1. [Install a Helm client](https://github.com/helm/helm#install) with a version higher than 2.10. Helm
```
Comment (frankbu) [inline content/en/docs/setup/install/helm/index.md]:
```suggestion
1. [Install a Helm client](https://github.com/helm/helm#install) with a version higher than 2.10.

    {{< warning >}}
    Helm versions `2.15.x` have a [known critical defect](https://github.com/helm/helm/issues/6708).
    This defect makes Helm 2.15 unsuitable for use with Istio.
    {{< /warning >}}
```

File: content/en/docs/setup/install/helm/index.md [hunk 2]
```diff
@@ -40,7 +40,9 @@ and then further customize the configuration for your specific needs.
...
+versions `2.15.z` have a [known critical defect](https://github.com/helm/helm/issues/6708). This defect
```
Comment (frankbu) [inline content/en/docs/setup/install/helm/index.md]:
```suggestion
```

File: content/en/docs/setup/install/helm/index.md [hunk 3]
```diff
@@ -40,7 +40,9 @@ and then further customize the configuration for your specific needs.
...
+makes Helm 2.15 unsuitable for use with Istio.
```
Comment (frankbu) [inline content/en/docs/setup/install/helm/index.md]:
```suggestion
```

**Comment by howardjohn** [comment]
We have so many warnings in our docs that make them more complicated. I would much rather we just workaround the issue than document it: https://github.com/istio/istio/pull/18170

**Comment by howardjohn** [comment]
https://github.com/helm/helm/releases/tag/v2.15.1

**Pull request closed without merging**